### ISSUE TITLE
Trying to extract table values from pdf

### ISSUE THREAD

**Issue by arunraja007** [tabulapdf/tabula#373]
http://usda.mannlib.cornell.edu/usda/waob/wasde//1980s/1989/wasde-12-12-1989.pdf  Need to extract tables from this pdf. But since table is embedded as image not able to get the values.

**Comment by hpiedcoq** [comment]
You can't extract data from pdf-images with Tabula. Github's project .md file is clear about that. It's not possible.

You can try to perform an OCR on the document using tesseract for example, or commercial products such as Adobe Acrobat Pro.
Depending on the quality of the origunal scan, ots resolution(I recommend 400DPI, no compression on PDF), you can get some good results. But it won't be perfect and you'll need to clean your data.

**Comment by hpiedcoq** [comment]
I've performed an OCR using NotablePDF ( https://tools.notablepdf.com/ocr)  on your document.

https://drive.google.com/file/d/0B1YVmQyWrFA8NTZxWmVnSW8zQ1k/view?usp=sharing

The quality is pretty good, and I've quickly tryied Tabula 0.9.7 and it works quite well.

**Comment by arunraja007** [comment]
Thanks a lot.. :)

**Comment by ozdogs** [comment]
Just tried Tabula for the first time and have played around and got some fairly good results.  Wonder if someone could help me with how to extract the individual table data from the attached file.  I have tried to manually select each small table and this worked OK, but is very time consuming as the tables are different sizes and in slightly different positions on each page and will vary from PDF file to another PDF file.  Tried the autoselect and the returned data was not separated into lines of data but all grouped together.  Would appreciate any help or suggestions.
[GEELONG 4 DEC.pdf](https://github.com/tabulapdf/tabula/files/2646523/GEELONG.4.DEC.pdf)
